pkg/gameserverallocations/find_test.go

```diff
@@ -127,7 +127,7 @@ func TestFindGameServerForAllocationPacked(t *testing.T) {
 				assert.Equal(t, ErrNoGameServer, err)
 				assert.Nil(t, gs)
 			},
-			features: fmt.Sprintf("%s=false&%s=false", runtime.FeaturePlayerAllocationFilter, runtime.FeatureStateAllocationFilter),
+			features: fmt.Sprintf("%s=false", runtime.FeaturePlayerAllocationFilter),
```

Comment:
Ditto here - remove the "false" cases (you can't disable it anymore)

Reply:
The expectation is that these lines will be removed https://github.com/googleforgames/agones/blob/main/pkg/gameserverallocations/find_test.go#LL96C6-L96C6 ?